javascript/CWE-094/CodeInjectionAudit.ql

```diff
@@ -0,0 +1,21 @@
+/**
+ * @name Audit: Usage of Code Injection sink
+ * @description Usage of Code Injection sink
+ * @kind problem
+ * @problem.severity error
+ * @security-severity 8.0
+ * @precision high
+ * @id js/audit/code-injection
+ * @tags security
+ *       external/cwe/cwe-094
+ *       external/cwe/cwe-095
+ *       external/cwe/cwe-079
+ *       external/cwe/cwe-116
+ *       audit
+ */
+
+import javascript
+import semmle.javascript.security.dataflow.CodeInjectionCustomizations
+
+from CodeInjection::Sink sinks
+select sinks, "user-provided value"
```

Comment:
What does this message mean? What value is user-provided? I think this message could do with being "Use of code injection sink", or something like that

Reply:
Completely agree, changed to "Code Injection sink"